packages/friends/yml/en/joseph-edgerton.yml

```diff
@@ -3,16 +3,16 @@ name: Joseph Edgerton
 slug: joseph-edgerton
 gender: male
 description:
-  ​Joseph Edgerton (1797-1875) was one of the few faithful Friends in the 19th century
-  that held fast to the Truth even as the long-favored Society of Friends crumbled to the
-  earth on the right hand and on the left. He was acknowledged a minister of the gospel in
-  1833, and travelled extensively in that capaity a great part of the time during the
-  subsequent years of his life. He was valiant for the Truth in his day, standing forth in
-  unflinching support of the original doctrines and testimonies of the Society of Friends.
-  The cleansing, purifying power of the Lord Jesus, in His inward and spiritual appearance
-  in the heart, was frequently the subject of his preaching, and his greatest desire was
-  that all would experience a growth in the true grace of God, and so be continually
-  "taking root downward, and bringing forth fruit upward."
+  Joseph Edgerton (1797-1875) was one of the few faithful Friends in the mid 19th century
+  that held fast to the Truth, even as the long-favored Society of Friends crumbled to
+  pieces on the right hand and on the left. He was acknowledged a minister of the gospel
+  in 1833, and travelled extensively in that capaity a great part of his life. He was
+  valiant for the Truth in his day, standing forth in unflinching support of the original
+  doctrines and testimonies of the Society of Friends. The cleansing, purifying power of
+  the Lord Jesus, in His inward and spiritual appearance in the heart, was frequently the
```

Comment:
Sometimes I think you put in too many commas, but I'm not sure enough of myself to call them all out. This sentence (starting with "The cleansing, purifying..." is one.  I think I would kill the one after `Jesus,` and after `in the heart,`.